client/style.sass

```diff
@@ -10,28 +10,34 @@ body
 
 #App
 	display: flex
-	flex-direction: row
+	flex-direction: column
 	width: 100%
 	height: 100%
 	background-color: library.$main-colour
 
 .logoButton
-	width: 15%
+	width: 200px
+
+.mainContainer
+	display: flex
+	flex-direction: column
+	justify-content: space-around
+	align-items: center
 
-.Main
-	width: 70%
+.main
 	display: flex
 	flex-direction: column
-	//justify-content: center
+	max-width: library.$max-width
 
-.AddButtons
-	width: 15%
 
-.PageButtons
+.addButtons
```

Comment:
Would `menuButtons` make more sense? I think of `addButtons` as including the floating ones.

Reply:
Im not actually sure what addButtons is, so maybe?
The change was mostly that you don't usually start css classes with capitals